Gcpe.Hub.API/Controllers/ActivitiesController.cs

```diff
@@ -30,101 +30,95 @@ public class ActivitiesController : ControllerBase
         }
 
         [HttpGet("Forecast/{numDays}")]
-        [Produces(typeof(IEnumerable<ActivityViewModel>))]
-        [ProducesResponseType(200)]
+        [Produces(typeof(IEnumerable<Models.Activity>))]
         [ProducesResponseType(400)]
         public IActionResult GetNewsForecast(int numDays)
         {
             try
             {
                 var today = new DateTime(2018, 9, 1);// DateTime.Today; // for testing
```

Comment:
I wonder if there is a better way to have this date for development. maybe we can use a feature flag in an environment variable.. just worried we will forget to set it back to the proper date.